README.md

```diff
@@ -1,68 +1,84 @@
-# Commento
+<h1 align="center">Commento</h1>
 
-An open source, lightweight, and tracking-free comment engine.
+<p align="center">A lightweight, open source, tracking-free comment engine alternative to Disqus.</p>
 
-![Example](https://cloud.githubusercontent.com/assets/7521600/25356132/d00013e0-2956-11e7-8dba-772a8040ae0c.png)
+<p align="center"><a href="https://adtac.github.io/commento/example/demo.html">
+Check out a live demo of Commento here.
+</a></p>
+
+<p align="center"><img src="https://cloud.githubusercontent.com/assets/7521600/25660780/6a15a546-302b-11e7-8b55-f200ff856797.png" alt="Example"></p>
 
 ### Installation
 
-**[I've hosted a live demo here. Check it out!](https://adtac.github.io/commento/example/demo.html)**
+You need to have an instance of Commento running in your server. Commento is written in Go so you'll need to [install Go first](https://golang.org/dl/). Once that's done, you can build the server. First get the source:
+
+```bash
+$ go get -v github.com/adtac/commento
+```
+
+Then go to the directory and run:
 
-It's really simple to embed a Commento section to your webpage. A trivial page would look like:
+```bash
+$ go build .
+```
+
+To start the server, run `./commento` from the build directory. By default the server will started on port 8080. If you want to change this, you can provide a environment variable. For example, if you want the server running on port `1234`:
```

Comment:
[https://peter.bourgon.org/go-in-production/](https://peter.bourgon.org/go-in-production/) Peter Bourgon here argues in favour of flags over env vars. I do tend to agree that `./commento --option=foo` is better than `export OPTION=foo` and `./commento`, since it's more explicit.